### PR TITLE
config : Add DESKTOP_SHARING_FRAMERATE_MIN and MAX Param

### PR DESCRIPTION
**Pull request by spprod35**
Add param for screensharing framerate to the new config file

### CODE REVIEW

File: web/rootfs/defaults/settings-config.js [hunk 1]
```diff
@@ -29,6 +29,8 @@
 {{ $DISABLE_AUDIO_LEVELS := .Env.DISABLE_AUDIO_LEVELS | default "false" | toBool -}}
 {{ $ENABLE_NOISY_MIC_DETECTION := .Env.ENABLE_NOISY_MIC_DETECTION | default "true" | toBool -}}
 {{ $START_VIDEO_MUTED := .Env.START_VIDEO_MUTED | default 10 -}}
+{{ $DESKTOP_SHARING_FRAMERATE_MIN := .Env.$DESKTOP_SHARING_FRAMERATE_MIN | default 5 -}}
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:32]: that $ after .Env is not right.

**Review by saghul — CHANGES_REQUESTED** [review]
You also need to add these to docker-compose.yml

**Comment by spprod35** [comment]
I made the changes, sorry for this $ error

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Thank you!

**Pull request merged**